tests/stdlib/tfdleak_multiple.nim

```diff
@@ -1,6 +1,7 @@
 import os, osproc, strutils
 
-const Iterations = 200
+const Iterations = when defined(windows) or defined(netbsd): 200 else: 1
```

Comment:
Either the same between all of them, or disable it entirely on non-Windows. It's pointless to run 1 extra iteration of the test since that show nothing at all.

Also, for NetBSD, it's a bug in their libc that will trigger the issue 100% of the time, so we don't need this flaky test for it.

Reply:
pushed a commit; it will now be megatest-joinable, on all platforms, but will only run the `testFdLeak()` code on windows

joinining megatest (but not calling `testFdLeak()`) is extremely cheap, so might as well do it; it removes chance for os-specific errors.

Reply:
oh, and the duplicate entry `tests/stdlib/tfdleak.nim` is because of `matrix: "; -d:nimInheritHandles"` ; but really that line should show the full cmdline instead of (or in addition to) abbreviated line